Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -470,6 +470,31 @@ final class UIViewExtensionsTests: XCTestCase {
         XCTAssertEqual(buttonSubview.ancestorView(withClass: UITableView.self), tableView)
     }
 
+    func testConstraintFinders() {
+        let container = UIView()
+        let view = UIView()
+        container.addSubview(view)
+
+        // setup constraints, some in container and some in view
```

Comment:
Can you please separate the test cases? Like one for all constraints, one for finding direct calls and one for computed properties?

Reply:
done